web/applications/metadata-editor/makeXML.php

```diff
@@ -108,7 +108,7 @@ function createNodesXML($xml, $doc, $validated)
                     $cdata = $doc->createCDATASection($val);
                     $node = $node->appendChild($cdata);
                 } else {
-                    //$val = htmlspecialchars($val, ENT_QUOTES | 'ENT_XML1', 'UTF-8');
+                    //$val = htmlspecialchars($val, ENT_QUOTES | ENT_XML1, 'UTF-8');
```

Comment:
We can remove this commented out code, but i think it was a find and replace

Reply:
It was left there on purpose, because MDE is so old.